pilot/prompts/components/files_list.prompt

```diff
@@ -1,6 +1,8 @@
-{% if files|length > 0 %}Here are files that are currently implemented:
+{% if file_summaries %}These files are currently implemented:{% for fpath, summary in file_summaries.items() %}
+* `{{ fpath }}`: {{ summary }}{% endfor %}
+{% endif %}{% if files|length > 0 %}Here are the relevant files:
 ---START_OF_FILES---{% for file in files %}
-**{{ file.path }}/{{ file.name }}** ({{ file.lines_of_code }} lines of code):
+**{% if file.path %}{{ file.path }}/{% endif %}{{ file.name }}** ({{ file.lines_of_code }} lines of code):
```

Comment:
This avoids turning files in the project root (like `app.js`) into `/app.js` because the path is empty.